implementations/test/ERC725X.behaviour.ts

```diff
@@ -855,7 +855,8 @@ export const shouldBehaveLikeERC725X = (
                 .withArgs(
                   txParams.Operation,
                   addressContractCreatedChecksumed,
-                  txParams.value
+                  txParams.value,
+                  "0x" + "0".repeat(64)
```

Comment:
Maybe we can use ethers instead for this?

```suggestion
                  ethers.utils.hexZeroPad("0x00", 32)
```